packer/scripts/vagrant.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+# install the insecure public key for Vagrant
+mkdir -p /home/vagrant/.ssh
+curl https://raw.githubusercontent.com/mitchellh/vagrant/master/keys/vagrant.pub > /home/vagrant/.ssh/authorized_keys
```

Comment:
I've added these in a push, build #13 should fix them: https://atlas.hashicorp.com/brianhicks/build-configurations/mi